src/web/views/index.js

```diff
@@ -41,6 +41,13 @@ const heapAnalytics = (config) => {
     </script>` : '';
 };
 
+const simpleAnalytics = (config) => {
+  return config.useAnalytics ?
+    `<script async defer data-collect-dnt="true" src="https://scripts.simpleanalyticscdn.com/latest.js"></script>
```

Comment:
The `data-collect-dnt="true"` means that if a user has "Do Not Track" enabled clientside, we will ignore it and collect data from them anyway. Is this something we want to do? CC @huslage